shared-libs/server-checks/src/checks.js

```diff
@@ -0,0 +1,76 @@
+var url = require('url'),
+    request = require('request');
+
+var MIN_MAJOR = 8;
+
+var nodeVersionCheck = (medicModule) => {
+  try {
+    var [major, minor, patch] = process.versions.node.split('.').map(Number);
+    if (major < MIN_MAJOR) {
+      throw new Error(`Node version ${major}.${minor}.${patch} is not supported, minimum is ${MIN_MAJOR}.0.0`);
+    }
+    console.log(`Node Environment Options: '${process.env.NODE_OPTIONS}'`);
+    console.log(`Node Version: ${major}.${minor}.${patch} in ${process.env.NODE_ENV || 'development'} mode`);
+  } catch (err) {
+    console.error(`Fatal error initialising medic-${medicModule}`);
```

Comment:
We don't need to pass in the name of the module - this will be logged in a log file which is named after the module.